advanced_source/cpp_custom_ops.rst

```diff
@@ -63,9 +63,45 @@ Using ``cpp_extension`` is as simple as writing the following ``setup.py``:
 
 If you need to compile CUDA code (for example, ``.cu`` files), then instead use
 `torch.utils.cpp_extension.CUDAExtension <https://pytorch.org/docs/stable/cpp_extension.html#torch.utils.cpp_extension.CUDAExtension>`_.
-Please see how
-`extension-cpp <https://github.com/pytorch/extension-cpp>`_ for an example for
-how this is set up.
+Please see `extension-cpp <https://github.com/pytorch/extension-cpp>`_ for an
+example for how this is set up.
+
+In PyTorch 2.6 and later, if your custom library adheres to the `CPython stable
```

Comment:
I think it would be good to add an extra sentence here to be very explicit about what this bring: 
```suggestion
Starting with PyTorch 2.6, you can now build a single wheel for all CPython version (like you would do for pure python packages). 
In particular, if your custom library adheres to the `CPython stable
```